Dockerfile-slim.template

```diff
@@ -1,6 +1,7 @@
 FROM buildpack-deps:jessie-curl
 
-RUN groupadd -r node && useradd -r -g node node
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --create-home node
```

Comment:
Using 1000 as UID in the container is the only viable fix mentioned multiple times in the referenced issue.

I really value simplicity too, but I think understandable code should be valued higher.
A code line which simply adds a user and has the implicit assumption that it will assign the UID 1000 has actually two issues:

1. The assigned UID might actually not be 1000, if there's already a user with that UID in the base image. Might never happen, but I also assumed there would never be a user named `node` in this NodeJS base image here but now there is and it was a breaking change for us.
2. It is not clear from reading the code line that this user is supposed to have the UID 1000. One needs an explicit understanding that the first (non-system) user created will have that ID and as explained in Number 1, might not always be true.

Anyway, I'm fine with the simpler code line without the check, I just think the check would add additional value.

But more important is that it's not a system user, but one with UID 1000 and a home directory.